### DISCUSSION TITLE
Welcome to pocketbase-async discussions!

### DISCUSSION THREAD

**Discussion by thijsmie** [thijsmie/pocketbase#1] [Announcements]
# 👋 Welcome!

This is the place to ask questions about `pocketbase-async`, share ideas and using pocketbase in python in general. If you're requesting features or found a bug please use an issue instead.

**Comment by matthieuEv** [comment]
When do you think a new release to Pypi could come?

**Comment by thijsmie** [comment]
I just pressed the button for you, should be up in a couple minutes!
